### PR TITLE
feat(workflow): Setup inbox layout

### PR DESCRIPTION
**Pull request by scttcper**
Behind flag `organization:inbox`, adds tabs and new header with full width search box.

image

### CODE REVIEW

File: src/sentry/static/sentry/app/components/queryCount.tsx [hunk 1]
```diff
@@ -37,9 +39,9 @@ const QueryCount = ({
 
   return (
     <div className={cx}>
-      <span>(</span>
+      {!hideParens && <span>(</span>}
```
Comment (taylangocmen) [inline src/sentry/static/sentry/app/components/queryCount.tsx:42]: Are we sure we want to do this?
When you swap to a tab and then do a custom search, the last tab you were on still has the count, which looks inconsistent.
I would prefer to have the counts visible always.

File: src/sentry/static/sentry/app/views/issueList/header.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        <StyledHeaderContent>
+          <StyledLayoutTitle>{t('Issues')}</StyledLayoutTitle>
+        </StyledHeaderContent>
+      </BorderlessHeader>
+      <TabLayoutHeader>
+        <Layout.HeaderNavTabs underlined>
+          {queries.map(([tabQuery, queryName]) => (
+            <li key={tabQuery} className={query === tabQuery ? 'active' : ''}>
+              <a onClick={() => onTabChange(tabQuery)}>
+                {queryName} {query === tabQuery && count}
```
Comment (taylangocmen) [inline src/sentry/static/sentry/app/views/issueList/header.tsx:38]: Are we sure we want to do this? In designs we have Inbox having the count as well as the active tab.

**Comment by taylangocmen** [comment]
Other than the tab counts lgtm but should wait for https://github.com/getsentry/sentry/pull/21684

File: src/sentry/static/sentry/app/views/issueList/header.tsx (hunk 2 above)
Comment (scttcper) [reply src/sentry/static/sentry/app/views/issueList/header.tsx:38]: we'd have to make another query to always show the count on the inbox, can take a look at that in another PR

Comment (taylangocmen) [reply src/sentry/static/sentry/app/views/issueList/header.tsx:38]: ok sounds good

**Review by taylangocmen — APPROVED** [review]
lgtm

**Pull request merged**